### ISSUE TITLE
Bug on the option to clear avatar it need adjust.

### ISSUE THREAD

**Issue by ghost** [almindor/harbour-jTox#67]
![captura_20180920_001](https://user-images.githubusercontent.com/24536606/45855226-193e1380-bd1d-11e8-9cf9-687bd2b0bd66.png)
#

**Comment by almindor** [comment]
Yes I'm aware. I decided to release anyway since it "works" even if it looks bad.

I need to find out which element in that page is "locking" the context menu from pushing things down. Sailfish implemented this in a way that some absolute/locking layout setups block the popup menu from pushing the rest of the items when it opens and it's not always clear what causes it.

Will try to fix by next release.

**Comment by carlosgonz0** [comment]
this is my issue report, any news? when new version openrepos? :  )

**Comment by almindor** [comment]
Currently busy with the `AV` branch trying to make audio calls work. This issue was pretty time consuming since I wasn't able to figure out why it doesn't push the bottom elements out when opening that popup menu.

It just isn't worth investigating further at this point. I'm not ignoring it but there are simply more important things to do with jTox. Any help is welcome of course!
